.github/workflows/test.yaml

```diff
@@ -0,0 +1,35 @@
+name: test
```

Comment:
Maybe a more descriptive name here, something with "suite" or "pytest" or something?  Here's what the checks interface currently looks like:

![image](https://user-images.githubusercontent.com/57452607/97749747-35443f80-1ab5-11eb-816d-a4e925082b9a.png)